java/src/jmri/UserPreferencesManager.java

```diff
@@ -551,102 +551,101 @@ public interface UserPreferencesManager {
 
      final UserPreferencesManager p;
      p = jmri.InstanceManager.getDefault(jmri.UserPreferencesManager.class);
-     if (p.getRouteSaveMsg()){
-     final JDialog dialog = new JDialog();
-     dialog.setTitle("Reminder");
-     dialog.setLocationRelativeTo(null);
-     dialog.setDefaultCloseOperation(javax.swing.JFrame.DISPOSE_ON_CLOSE);
-     JPanel container = new JPanel();
-     container.setBorder(BorderFactory.createEmptyBorder(10,10,10,10));
-     container.setLayout(new BoxLayout(container, BoxLayout.Y_AXIS));
-
-     JLabel question = new JLabel("Remember to save your Route information.", JLabel.CENTER);
-     question.setAlignmentX(Component.CENTER_ALIGNMENT);
-     container.add(question);
-
-     JButton okButton = new JButton("Okay");
-     JPanel button = new JPanel();
-     button.setAlignmentX(Component.CENTER_ALIGNMENT);
-     button.add(okButton);
-     container.add(button);
-
-     final JCheckBox remember = new JCheckBox("Do not remind me again?");
-     remember.setAlignmentX(Component.CENTER_ALIGNMENT);
-     remember.setFont(remember.getFont().deriveFont(10f));
-     container.add(remember);
-
-     okButton.addActionListener(new ActionListener(){
-     public void actionPerformed(ActionEvent e) {
-     if(remember.isSelected()){
-     p.setRouteSaveMsg(false);
-     }
-     dialog.dispose();
-     }
-     });
-
-
-     dialog.getContentPane().add(container);
-     dialog.pack();
-     dialog.setModal(true);
-     dialog.setVisible(true);
+     if (p.getProperty("thisClass", "routeSaveMsg") { // NOI18N
+         final JDialog dialog = new JDialog();
+         dialog.setTitle("Reminder"); // I18N
+         dialog.setLocationRelativeTo(null);
+         dialog.setDefaultCloseOperation(javax.swing.JFrame.DISPOSE_ON_CLOSE);
+         JPanel container = new JPanel();
+         container.setBorder(BorderFactory.createEmptyBorder(10,10,10,10));
+         container.setLayout(new BoxLayout(container, BoxLayout.Y_AXIS));
+
+         JLabel question = new JLabel("Remember to save your Route information.", JLabel.CENTER);
+         question.setAlignmentX(Component.CENTER_ALIGNMENT);
+         container.add(question);
+
+         JButton okButton = new JButton("Okay");
+         JPanel button = new JPanel();
+         button.setAlignmentX(Component.CENTER_ALIGNMENT);
+         button.add(okButton);
+         container.add(button);
+
+         final JCheckBox remember = new JCheckBox("Do not remind me again");
```

Comment:
Should this be using the new NamedBeanBundle String DontRemind ?

Reply:
Yes that would be better. Will change that